docker-compose.yml

```diff
@@ -110,3 +110,9 @@ services:
       - -Ctransport.host=localhost
     environment:
       - CRATE_HEAP_SIZE=128m
+  supervisor:
+    image: bravado/supervisord
```

Comment:
this doesn't appear to be an official image, nor does there appear to be one. Why did you pick this one?

Reply:
Supervisor doesn't have any official images, but it can be installed with package manager in almost any mainstream distro. Integration tests needs to be almost completely revorked since this iamge is actually not classic supervisord, but some golang implementation of it.